book/01-introduction/sections/about-version-control.asc

```diff
@@ -23,7 +23,7 @@ To deal with this issue, programmers long ago developed local VCSs that had a si
 image::images/local.png[Local version control diagram]
 
 One of the more popular VCS tools was a system called RCS, which is still distributed with many computers today.
-Even the popular Mac OS X operating system includes the `rcs` command when you install the Developer Tools.
+Even the popular Apple macOS operating system includes the `rcs` command when you install the Developer Tools.
```

Comment:
Actually, this doesn't seem to be true anymore:

```bash
bash-3.2$ rcs
bash: rcs: command not found
```

Could you remove this sentence?

Reply:
Done.